lib/models/Message.js

```diff
@@ -109,6 +109,45 @@ export default class Message extends Model {
   }
```

Comment:
Do we provide an `update` method there, it would allow to make a batch of flag changes for example.